### ISSUE TITLE
AOS - enhance Running an AOS guide to add connecting to an AGC running in a Docker container?

### ISSUE THREAD

**Issue by jay-cousins** [autonity/docs.autonity.org#101]
### Description

PR #100 adds how to guides for installing and running AOS as a Docker.

An enhancement could be to add an info box on how to connect to an AGC running in Docker - i.e.

- run `docker network ls` to get the docker network's running and the NAME
- run `docker network inspect <NAME>` to inspect the network and view info on the container running the AGC to get the WS 
-  IP address the node is running at
- specify that WS Address in the docker command for running AOS

### Rationale

- A user running the AOS as Docker may also be running the AGC as Docker

### Implementation

- Add an *Info* note box with a bullet point list of the steps to get and specify the  AGC Docker container's WS Address. See **Description** above.

**Comment by jay-cousins** [comment]
@raj-shekhar1 - dis/agree?

**Comment by raj-shekhar1** [comment]
@cmjc Yes, docs should provide snippets for this, the above mentioned approach of using docker internal IP to connect with ws endpoint is fine for testing but not reliable as internal IP can change on docker restart. I will provide a sample to add in the docs using `docker networks` which is cleaner.
Till then, it is safe to assume that user knows how to run docker containers using docker networks and *info* box can mention that 
```
 AOS requires an accessible `ws/wss` AGC endpoint. If you running AGC in docker and facing issues in connecting AOS  then check [link](https://docs.docker.com/network/).  
```

**Comment by jay-cousins** [comment]
Thanks @raj-shekhar1 I have edited PR #100 to add this as an info box, commit cd148589507fe6b41f9cf29031563431b2bfa868
